web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -1394,9 +1395,11 @@ private BaseQueryResponse _next(RunningQuery query, String queryId, Collection<S
         
         query.getMetric().setProxyServers(proxyServers);
         
-        testForUncaughtException(query.getSettings(), resultList);
+        testForUncaughtException(query.getSettings(), resultsPage);
         
-        if (resultList.getResults().isEmpty()) {
+        // This should NOT be an exception - revisit. Unfortunately, the jboss interceptor looks for a
```

Comment:
create a ticket?

Reply:
This will go away in the query microservices, so I don't think a ticket is worth it here.